schema/caveschema.xsd

```diff
@@ -1,6 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="unqualified" attributeFormDefault="unqualified">
```

Comment:
My linter says that `elementFormDefault="unqualified" attributeFormDefault="unqualified"` are both redundant (are those the default values?). I'm not sure if that's true in this case. 
```suggestion
<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema">
```

Reply:
I'm not super sure about these ones so perhaps I'll leave them? They seem to be some overall configuration setup that I don't want to mess up

Reply:
I believe the default value is `none`, though `unqualified` and `none` seem to do similar things

https://docs.microsoft.com/en-us/dotnet/api/system.xml.schema.xmlschema.attributeformdefault?view=net-6.0